Jenkinsfile

```diff
@@ -305,12 +305,12 @@ def buildPerformance() {
           sh 'unzip core-benchmarks.zip -d test/bench/'
           sh 'rm core-benchmarks.zip'
 
-          sh '''
+          sh """
             cd test/bench
             mkdir -p core-benchmarks results
             ./gen_bench_hist.sh
             ./parse_bench_hist.py --local-html results/ core-benchmarks/
-          '''
+          """
```

Comment:
Sorry, this is Groovy so the `"""` and `'''` are called weak and strong as in shell. The correct term is _interpolated_, but the idea is the same.